cart/world.go

```diff
@@ -0,0 +1,28 @@
+package cart
+
+import "cart/tic80"
+
+type World struct {
+	X, Y int32
+}
+
+func NewWorld(player *Player) World {
+	tileX, tileY := worldPosition(player)
+	return World{tileX, tileY}
+}
+
+func worldPosition(player *Player) (int32, int32) {
+	playerTileX := (player.X / 8) - (tic80.SCREEN_TILE_WIDTH / 2)
```

Comment:
We need to decide whether we want smooth player movement or block-by-block movement

Reply:
@fjebaker seems to be implementing smooth movement right now